src/r2mm/manager/PackageDexieStore.ts

```diff
@@ -123,6 +126,40 @@ export async function getPackagesByNames(community: string, packageNames: string
     return packages.map(ThunderstoreMod.parseFromThunderstoreData);
 }
 
+/**
+ * @param community Community which package listings should be used in the lookup.
+ * @param dependencies Lookup targets as Thunderstore dependency strings.
+ * @param useLatestVersion Ignore the version number in dependencyString and return the latest known version.
+ * @returns ThunderstoreCombo[], silently omitting unknown packages and versions.
+ */
+export async function getCombosByDependencyStrings(
+    community: string,
```

Comment:
It would be better for this to take a `Game` type and for this method to handle how to extract that. I don't think that the download provider should need to know that an internal folder name is what drives the community identifier.

Reply:
Makes sense.

For the background, when I whipped up the Dexie schema, I would've preferred to use the same community slug as is used in Thunderstore, but it wasn't available (aside from stripping from the API URL). I didn't want to add a new field to the GameManager mega list which already contains various formats of the spelling the game's name, so I just picked one that seemed most suitable. So although the variable is named "internalFolderName", what I use it for is "community identifier".

Reply:
Yeah I'm happy for it to be the community identifier, it's mostly just to separate functional concerns

Reply:
Changed to accept [Game object instead of id string](https://github.com/ebkr/r2modmanPlus/pull/1482/commits/9d71ae7c907b9b0ca24911a070847b4cc03b4974#diff-2122463b05e5c9342979622ca58d48970d706008d1d9df423063e69e07055988R137).